ddtrace/internal/datadog/profiling/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.19)
+
+# We're building a Python extension and expect to be called from setup.py
+set(EXTENSION_NAME "" CACHE STRING "Name of the extension")
```

Comment:
Is this a placeholder, or should this be something like "datadog profiling"

Reply:
There's actually an elaborate reason for this.  I updated the comment to better reflect the convention imposed on us by our setup fixture.